src/socket/index.ts

```diff
@@ -80,6 +80,13 @@ function socket({
       }
     });
 
+    socket.on('message', ({ content, to }) => {
+      socket.to(to).emit('message', {
+        content,
+        from: socket.id
+      })
```

Comment:
ini better ngirimnya ke roomId dari data socket yg disimpen, jd kaya `io.to(socket.data.roomId).emit(...)` jadi params `{ to }` nya bisa diilangin aja. Tujuannya biar messagenya dikirim ke kedua user di room tersebut, nanti parameter `from` nya bisa dipake di FE buatnnetuin itu dikirim orang lain atau diri sendiri